lib/autokey/qtui/dialogs/windowfiltersettings.py

```diff
@@ -48,7 +48,11 @@ def load(self, item: model.Item):
             self.apply_recursive_check_box.setChecked(item.isRecursive)
 
     def save(self, item):
-        item.set_window_titles(self.get_filter_text())
+        try:
+            item.set_window_titles(self.get_filter_text())
+        except re.error:
+            # TODO: Warn user. Currently just doesn't save regex on error.
+            pass
```

Comment:
I thought a bit about it. For now, just logging this as a warning in all occurrences should be sufficient.